inst/rmarkdown/templates/isba/skeleton/sample.bib

```diff
@@ -0,0 +1,2244 @@
+%% BiBTeX database for models.tex bibliography
```

Comment:
Do we use all references ? Is it possible to reduce the sample bib file to the bare minimum. Aim is to save space.

Reply:
I will slim this to 3 records.